packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -107,6 +113,7 @@ export class DataStores implements IDisposable {
         baseLogger: ITelemetryBaseLogger,
         getDataStoreBaseGCDetails: () => Promise<Map<string, IGarbageCollectionSummaryDetails>>,
         private readonly dataStoreChanged: (id: string) => void,
+        private readonly rootDataStoreAdded: (handle: IFluidHandle) => void,
```

Comment:
Is there any way not to make root data stores special? I.e. semantically manage it via "path was added to node", where we do have root "/" node (root of everything in container)?

Reply:
That makes sense. I will update it.